client/src/components/Listings/ListingCard/ListingCardCarousel/ListingCardCarouselElements.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+import Carousel from 'react-bootstrap/Carousel';
+
+export const ListingCardCarousel = styled(Carousel)`
+  /* position: absolute; */
+  width: 100%;
+  height: 100%;
+
+  .carousel-item {
+    transition: transform .4s ease-in-out;
+  /* top: 50px; */
+  }
+

```

Comment:
Remove commented out code.
```suggestion
  width: 100%;
  height: 100%;

  .carousel-item {
    transition: transform .4s ease-in-out;
  }
```